.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
```

Comment:
Do we want something like this ?

Actually I can't get it to work yet for some reason...

Reply:
Actually this file needs to be added in the default branch of the repo.

@stechu @grenade